src/WatcherCommand.php

```diff
@@ -38,9 +41,26 @@ protected function determineOptions(InputInterface $input): array
         return $options;
     }
 
+    protected function getOptionsConfigFile()
```

Comment:
I feel that this method can be greatly simplified.

Rename this to `getOptionsFromFile`, add an `array` return typehint.